README.md

```diff
@@ -128,11 +128,12 @@ Here are some of the things coming soon:
 
   - [x] Support `$PEBBLE_SOCKET` and default `$PEBBLE` to `/var/lib/pebble/default`
   - [x] Define and enforce convention for layer names
-  - [ ] Dynamic layer support over the API
+  - [x] Dynamic layer support over the API
+  - [x] Configuration retrieval commands to investigate current settings
+  - [x] Status command that displays active services and their current status
+  - [ ] General system modification commands (writing configuration files, etc)
+  - [ ] Improve signal handling, e.g., sending SIGHUP to a service
   - [ ] Terminate all services before exiting run command
```

Comment:
We should do this one soon and before we tackle the custom sending of signals, since it affects every single user of Pebble.

Reply:
It's unclear to me that the workload processes *should* stop if the service manager process (Pebble) dies. Isn't it good that they keep going? Or is what you're suggesting just good cleanup behavior and/or how other service managers work?

Reply:
In the best scenario, we get a SIGTERM in k8s, and politely terminate all the services we're running. In the worst case, the container is abruptly terminated. In both of these, though, the underlying services are going away.

Another perspective: right now we have no way to keep track of services unless we're running. If Pebble gets restarted and we run "pebble services", what do we get?

Yet another perspective: ideally we'd run services in a way that if Pebble dies abruptly, services die as well. Otherwise we'll lose control of these services for the reason above. Not an issue for now since k8s will collect for us regardless, but once Pebble is running standalone more often we should fix that too.